myapp/api/urls.py

```diff
@@ -17,4 +21,8 @@
    path('permissions/', views.PermissionsView)
 ]
 
-urlpatterns += router.urls
\ No newline at end of file
+urlpatterns += router.urls
+
+if settings.DEBUG:
```

Comment:
Why is this only if debug is true?

Reply:
I just changed it, this was an mistake I forgot to delete when I was exploring different image storing options and using django for testing